software/moltres/wiki/input_example.md

```diff
@@ -432,6 +458,17 @@ on the variable back into the Jacobian used for Newton-Raphson. A more in-depth
 description of the material along with its relatives is given
 [here](http://mooseframework.org/wiki/PhysicsModules/PhaseField/DevelopingModels/FunctionMaterials/).
 
+The user can exert his/her greatest influence on the calculations through the
+`Materials` block. Increasing values of `rho` or `cp` will increase materials
+ability to store heat, enabling a greater reactor power. Because of application
+of an insulating boundary condition at the outer reactor wall in this example,
+modifying the thermal conductivity `k` has only a small impact on the
+simulation. However, if the conductivity is lowered by several orders of
+magnitude, the user will observe increases in radial gradients between the fuel
+channels and graphite and a decrease in reactor power. Use of different
```

Comment:
perhaps "as well as a decrease" ( just to separate this clause from the clause beginning with 'between')